model/src/main/java/org/projectnessie/model/Reference.java

```diff
@@ -50,7 +51,7 @@
     })
 @JsonSubTypes({@Type(Branch.class), @Type(Tag.class), @Type(Detached.class)})
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY, property = "type")
-public interface Reference extends Base {
+public interface Reference extends Base, Serializable {
```

Comment:
Sorry, I didn't get this comment. What I should do ?

Reply:
In some places you add a `serialVerionUid` but not in others. Since the classes are only serialized, and only guaranteed to be serializable w/o conflicts - aka no backwards compatiblity, it's not necessary to add `serialVerionUid`s.